src/test/java/io/reactivex/maybe/MaybeTest.java

```diff
@@ -826,6 +827,53 @@ public void accept(Disposable v) throws Exception {
         .assertFailure(TestException.class);
     }
 
+    @Test(expected = NullPointerException.class)
```

Comment:
Please put these unit test into their own files and also verify the cases when the onTerminate callback fail.

Reply:
Implemented